dsl/src/commonMain/kotlin/it/unibo/collektive/aggregate/api/Aggregate.kt

```diff
@@ -63,6 +63,25 @@ interface Aggregate<ID : Any> {
      */
     fun <Initial, Return> repeating(initial: Initial, transform: YieldingScope<Initial, Return>): Return
 
+    /**
+     * Observes the value of an expression [local] across neighbours.
+     *
+     * ## Example
+     *
+     * ```kotlin
+     * val field = neighboring(0)
+     * ```
+     *
+     * The field returned has as local value the value passed as input (0 in this example).
+     *
+     * ```kotlin
+     * val field = neighboring({ 2 * 2 })
+     * ```
+     *
+     * In this case, the field returned has the result of the computation as local value.
```

Comment:
In this case a Field of `() -> Int` is returned, right?

Reply:
Then the documentation sounds incorrect, the field returned has the computation (and not the result of the computation) as result.